server/sonar-server/src/main/java/org/sonar/server/project/ws/BulkDeleteAction.java

```diff
@@ -67,20 +71,31 @@ public void define(WebService.NewController context) {
       .createParam(PARAM_KEYS)
       .setDescription("List of project keys to delete")
       .setExampleValue(KEY_PROJECT_EXAMPLE_001);
+
+    support.addOrganizationParam(action);
   }
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    userSession.checkLoggedIn().checkIsRoot();
+    userSession.checkLoggedIn();
+
     List<String> uuids = request.paramAsStrings(PARAM_IDS);
     List<String> keys = request.paramAsStrings(PARAM_KEYS);
+    String orgKey = request.param(ProjectsWsSupport.PARAM_ORGANIZATION);
 
-    DbSession dbSession = dbClient.openSession(false);
-    try {
+    try (DbSession dbSession = dbClient.openSession(false)) {
+      final OrganizationDto org;
```

Comment:
move lines 87 to 94 to a function and get rid of the ugly variable declaration :)